fixtures/etc/fos4PL8.json

```
@@ -0,0 +1,368 @@
+        "color": "Cyan"
+      }
+    },
+    "Blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "Indigo": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Indigo"
+      }
+    },
+    "Strobe": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 0],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        },
+        {
+          "dmxRange": [1, 40],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Strobe",
+          "speedStart": "1Hz",
+          "speedEnd": "40Hz",
+          "comment": "Dark Strobe"
+        },
+        {
+          "dmxRange": [41, 80],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Strobe",
+          "speedStart": "1Hz",
+          "speedEnd": "40Hz",
+          "comment": "Bright Strobe"
+        },
+        {
+          "dmxRange": [81, 120],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Pulse"
+        },
+        {
+          "dmxRange": [121, 160],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Pulse",
+          "comment": "Random"
+        },
+        {
+          "dmxRange": [161, 200],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Spikes",
+          "comment": "Flicker"
+        },
+        {
+          "dmxRange": [201, 240],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        },
+        {
+          "dmxRange": [241, 254],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Closed"
+        },
+        {
+          "dmxRange": [255, 255],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Open"
+        }
+      ]
+    },
+    "Curve": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 9],
+          "type": "Generic",
+          "comment": "Incandescent"
+        },
+        {
+          "dmxRange": [10, 19],
+          "type": "Generic",
+          "comment": "Linear"
+        },
+        {
+          "dmxRange": [20, 29],
+          "type": "Generic",
+          "comment": "Incandescent Red Shift"
+        },
+        {
+          "dmxRange": [30, 39],
+          "type": "Generic",
+          "comment": "Linear Red Shift"
+        },
+        {
+          "dmxRange": [40, 255],
+          "type": "NoFunction",
+          "comment": "Reserved"
+        }
+      ]
+    },
+    "Fan": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 9],
+          "type": "Maintenance",
+          "comment": "Auto"
+        },
+        {
+          "dmxRange": [10, 19],
+          "type": "Maintenance",
+          "comment": "Off"
+        },
+        {
+          "dmxRange": [20, 20],
+          "type": "Maintenance",
+          "comment": "Slow"
+        },
+        {
+          "dmxRange": [21, 248],
+          "type": "Maintenance",
+          "comment": "Linear Increase In Speed"
```

Please use `"parameterStart": "slow", "parameterEnd": "fast"` here.

```suggestion
          "parameterStart": "slow",
          "parameterEnd": "fast"
```